tests/sources/template.py

```diff
@@ -0,0 +1,32 @@
+"""
+template.py
+
+An ansible-events event source plugin template.
+
+Arguments:
+    - delay: seconds to wait between events
```

Comment:
@benthomasson Can any other others be passed into the source plugin? How would the user specify the,

Reply:
@benthomasson Should all sources have a name, version, description section. A info or metadata method that can be invoked by the engine as it loads up the source plugins.

Reply:
Arguments can be anything that is valid YAML.

Reply:
Arguments are passed as a dictionary to the source plugin.   If it is a valid dictionary it can be passed.   The keys are not restricted.

Reply:
Yes, that metadata is a good idea.   I have not needed it yet so I have not implemented it.    We could use the python standards for modules or Ansible standards.   I am fine with either.